src/LibEvLoop.php

```diff
@@ -22,12 +23,14 @@ class LibEvLoop implements LoopInterface
     private $readEvents = [];
     private $writeEvents = [];
     private $running;
+    private $signals;
 
     public function __construct()
     {
         $this->loop = new EventLoop();
         $this->futureTickQueue = new FutureTickQueue();
         $this->timerEvents = new SplObjectStorage();
+        $this->signals = new Pcntl($this);
```

Comment:
`LibEvLoop` should use the signal handling provided by the extension, as it is more reliable and integrates with the loop backend being used by the extension. Same goes for `LibEventLoop` and `ExtEventLoop`.